lib/Controller/Campaign.php

```diff
@@ -609,6 +617,7 @@ public function preview($campaignId)
         {
             $duration += $layout->duration ;
             $extendedLayouts[] = ['layout' => $layout,
+                                  'duration' => gmdate('H:i:s', $layout->duration),
```

Comment:
This formatting should really be the responsibility of the View and not the Controller (because it is display formatting). You can do that in your Twig file by using the `|date` filter on L16 of `views/campaign-preview.twig`